composer.json

```diff
@@ -1,41 +1,46 @@
 {
-    "name":         "knplabs/knp-gaufrette-bundle",
-    "type":         "symfony-bundle",
-    "description":  "Allows to easily use the Gaufrette library in a Symfony project",
-    "keywords":     ["file", "filesystem", "media", "abstraction"],
-    "homepage":     "http://knplabs.com",
-    "license":      "MIT",
-    "authors":      [
-        {
-            "name":     "Antoine Hérault",
-            "email":    "antoine.herault@gmail.com"
-        },
-        {
-            "name":     "The contributors",
-            "homepage": "https://github.com/knplabs/KnpGaufretteBundle/contributors"
-        }
-    ],
-    "require":      {
-        "php": ">=7.1",
-        "knplabs/gaufrette": "^0.11",
-        "symfony/config": "^4.4|^5.0|^6.0",
-        "symfony/dependency-injection": "^4.4|^5.0|^6.0",
-        "symfony/http-kernel": "^4.4|^5.0|^6.0"
+  "name": "knplabs/knp-gaufrette-bundle",
```

Comment:
all these changes in spaces need to be reverted

Reply:
That's irrelevant, this PR is not about changing the format of the file.
I understand indeed that the format is probably not the most standard possible, but it should be changed in a different PR.